### PR TITLE
MEN- 2147 improved keep parameter handling

### PR DESCRIPTION
**Pull request by DominikAdamski**
If user set --keep param for conversion script, then intermediate files are saved, otherwise they are removed.

MEN-2147

### CODE REVIEW

File: mender-convert.sh [hunk 1]
```diff
@@ -332,6 +332,11 @@ do_make_sdimg_raspberrypi3() {
   sudo install -d -m 755 ${sdimg_primary_dir}/data
 
   set_fstab $device_type
+
+  [[ $keep -eq "-k" ]] && { rm -rf $output_dir/boot.vfat\
+     $output_dir/cmdline.txt  $output_dir/config.txt\
```
Comment (apodogrocki) [inline mender-convert.sh]: In this place there are no cmdline.txt & config.txt files. You deal with them in stage 5 for RP PI3.

File: rpi3-convert-stage-5.sh [hunk 2]
```diff
@@ -180,6 +181,9 @@ do_install_bootloader() {
 
   detach_device_maps ${sdimgmappings[@]}
 
+  [[ $keep -eq 0 ]] && { rm -rf ${output_dir}/config.txt ${output_dir}/cmdline.txt
```
Comment (apodogrocki) [inline rpi3-convert-stage-5.sh]: Separate **rm** commands with **;** sign. No need to use **recursive** for removing files.

File: mender-convert.sh (hunk 1 above)
Comment (DominikAdamski) [reply mender-convert.sh]: config.txt and cmdline.txt are also copied to output directory in stage 2 for RPi3.

File: rpi3-convert-stage-5.sh (hunk 2 above)
Comment (DominikAdamski) [reply rpi3-convert-stage-5.sh]: Done

File: mender-convert.sh (hunk 1 above)
Comment (apodogrocki) [reply mender-convert.sh]: Right!

File: mender-convert.sh [hunk 3]
```diff
@@ -332,6 +332,11 @@ do_make_sdimg_raspberrypi3() {
   sudo install -d -m 755 ${sdimg_primary_dir}/data
 
   set_fstab $device_type
+
+  [[ $keep -eq "-k" ]] && { rm -rf $output_dir/boot.vfat\
```
Comment (kacf) [inline mender-convert.sh]: `-eq` is an arithmetic operator, I don't think it will work with a "-k" argument. Use `=` instead.

Comment (DominikAdamski) [reply mender-convert.sh]: Done

**Pull request merged**